WcaOnRails/lib/solve_time.rb

```diff
@@ -128,10 +128,14 @@ def unskipped?
     !skipped?
   end
 
-  def completed?
+  def complete?
```

Comment:
@jonatanklosko, I chose to rename this from `completed?` to `complete?` because `incomplete?` sounds nicer than `incompleted?`.